### PR TITLE
Hacktoberfest: add project euler for problem 205

### PR DESCRIPTION
**Pull request by Takosaga**
### **Describe your change:**
Retrying with pr with problem 205
Second PR Hacktoberfest, getting better at this

* [x] Add an algorithm?
* [ ] Fix a bug or typo in an existing algorithm?
* [ ] Documentation change?

### **Checklist:**
* [x] I have read [CONTRIBUTING.md](https://github.com/TheAlgorithms/Python/blob/master/CONTRIBUTING.md).
* [x] This pull request is all my own work -- I have not plagiarized.
* [x] I know that pull requests will not be merged if they fail the automated tests.
* [x] This PR only changes one algorithm file.  To ease review, please open separate PRs for separate algorithms.
* [ ] All new Python files are placed inside an existing directory.
* [x] All filenames are in all lowercase characters with no spaces or dashes.
* [x] All functions and variable names follow Python naming conventions.
* [x] All function parameters and return values are annotated with Python [type hints](https://docs.python.org/3/library/typing.html).
* [x] All functions have [doctests](https://docs.python.org/3/library/doctest.html) that pass the automated testing.
* [x] All new algorithms have a URL in its comments that points to Wikipedia or other similar explanation.
* [x] If this pull request resolves one or more open issues then the commit message contains `Fixes: #{$ISSUE_NO}`.

### CODE REVIEW

File: project_euler/problem_205/sol1.py [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+        for num, sum_2 in enumerate(sums_four_sided):
+            if sum_2 > sum_1:
+                count += len(sums_four_sided) - num
+                break
+
+    return float(format(count / (len(sums_six_sided) * len(sums_four_sided)), ".7"))
+
+
+if __name__ == "__main__":
+    print(f"{solution() = }")
```
Comment (xcodz-dot) [inline project_euler/problem_205/sol1.py:58]:
```suggestion
    print(f"{solution() = }")

```
PEP 8 new line fix

**Review by poyea — APPROVED** [review]
Thank you for your pull request!🤩

File: project_euler/problem_205/sol1.py (hunk 1 above)
Comment (poyea) [inline project_euler/problem_205/sol1.py:58]:
```suggestion
    print(f"{solution()}")
```

**Comment by dhruvmanila** [comment]
Hi @Takosaga your solution is taking more than 4 minutes to run. The pull request has been flagged to require optimization. Please update the solution accordingly. If you need any help, feel free to ask.

https://github.com/TheAlgorithms/Python/pull/3688/checks?check_run_id=1735382752#step:5:16

File: project_euler/problem_205/sol1.py (hunk 1 above)
Comment (dhruvmanila) [reply project_euler/problem_205/sol1.py:58]: `print(f"{solution() = }")` 

This is syntactically correct. This was introduced in Python 3.8: https://docs.python.org/3/whatsnew/3.8.html#f-strings-support-for-self-documenting-expressions-and-debugging

**Pull request closed without merging**